07-documenting_R.Rmd

```diff
@@ -10,11 +10,11 @@ R’s documentation has long been considered to be good for a free programming l
 
 The involvement of the community takes many forms, from authoring to bug reports to raising an issue when the documentation could be more complete or easier to use.
 
-This chapter is aimed at authors and potential authors of documentation for R. More specifically, it is for people contributing to the standard documentation and developing additional documents using the same tools as the standard documents. Any time you feel that you can clarify existing documentation or provide documentation that is missing, your contribution will be welcome and highly appreciated. If you find it difficult to deal with the markup language, you can ask for help for that part too. Please do not let the material in this chapter stand between the documentation and your desire to help out.
+This chapter is aimed at authors and potential authors of documentation for R. More specifically, it is for people contributing to the standard documentation and developing additional documents using the same tools as the standard documents. Any time you feel that you can clarify existing documentation or provide documentation that is missing, your contribution is welcome and appreciated. If you find it difficult to deal with the markup language, you can ask for help for that part too. Please do not let the material in this chapter stand between the documentation and your desire to help out. However, not every good faith effort to to change or extend the documentation will be accepted - sometimes the patch contribution might be incorrect, other times, while a change in wording may make some things clearer, other finer details of the corner cases or the like may be less clear, leading to the patch being declined or modified by the R-core member reviewing it before applying it (if they agree the issue is important enough to fix).
 
 ## Guidelines for writing the system help files (in .Rd format)
 
-Following are the suggested guidelines while creating the system help files in the .Rd format. These are intended for writing the core documentations but may also be useful for package writers. (share documentation of a simple base function here?)
+Following are the suggested guidelines while creating the system help files in the .Rd format. A example of the .Rd format is available for [mean.Rd](https://svn.r-project.org/R/trunk/src/library/base/man/mean.Rd). These are intended for writing the core documentations but may also be useful for package writers. Extensive details of writing R documentation files can be found [here](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Writing-R-documentation-files).
```

Comment:
Just a suggestion: Instead of referring to an external resource, you could also copy a significant portion (or all) of that as a code block and then when explaining the components later on in a bit more detail, point to the sections in the example. This would make for a bit more self-contained reading.

Reply:
I agree that it would be good to mention roxygen and that this creates .Rd files but that this guide focuses on .Rd files only because the man pages for the base distribution are written directly as .Rd files.